src/mobu/models/business.py

```diff
@@ -0,0 +1,85 @@
+"""Models for monkey business."""
```

Comment:
I clearly need to learn something about Pydantic.  Models look a lot like dataclasses to me.  Ah, I see, you can sort of substitute dataclasses for BaseModels: https://pydantic-docs.helpmanual.io/usage/dataclasses/

Reply:
Pydantic models are a lot like dataclasses except they have validation support, which is necessary for FastAPI.